libbeat/common/cfgwarn/removed.go

```diff
@@ -0,0 +1,48 @@
+package cfgwarn
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/joeshaw/multierror"
+)
+
+func CheckRemoved5xSettings(cfg *common.Config, settings ...string) error {
```

Comment:
I named it 5x so we can remove all references to 5.x at once (Just delete the function and compiler will tell you all places you have to modify).